modules/react/button/lib/PrimaryButton.tsx

```diff
@@ -40,6 +46,9 @@ export interface PrimaryButtonProps extends Themeable, GrowthBehavior {
   children?: React.ReactNode;
 }
 
+// Button sizes where data labels are enabled
+const dataLabelSizes = ['medium', 'large'];
```

Comment:
`dataLabelSizes` -> `dataLabelSize` to match `ButtonSize`

Reply:
I would argue plural here as there are multiple label sizes in the array.

Reply:
ok!